src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/Models/Database.cs

```diff
@@ -0,0 +1,305 @@
+// 
+// Code generated by Microsoft (R) AutoRest Code Generator 0.17.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.Management.Sql.Models
+{
+    using System.Linq;
+
+    /// <summary>
+    /// Represents an Azure SQL Database.
+    /// </summary>
+    [Microsoft.Rest.Serialization.JsonTransformation]
+    public partial class Database : Resource
+    {
+        /// <summary>
+        /// Initializes a new instance of the Database class.
+        /// </summary>
+        public Database() { }
+
+        /// <summary>
+        /// Initializes a new instance of the Database class.
+        /// </summary>
+        /// <param name="location">Resource location</param>
+        /// <param name="name">Resource name</param>
+        /// <param name="id">Resource Id</param>
+        /// <param name="type">Resource type</param>
+        /// <param name="tags">Resource tags</param>
+        /// <param name="collation">The collation of the Azure SQL
+        /// Database.</param>
+        /// <param name="creationDate">The creation date of the Azure SQL
+        /// Database (ISO8601 format).</param>
+        /// <param name="containmentState">The containment state of the Azure
+        /// SQL Database.</param>
+        /// <param name="currentServiceObjectiveId">The current Service Level
+        /// Objective Id of the Azure SQL Database. This is the Id of the
+        /// Service Level Objective that is currently active.</param>
+        /// <param name="databaseId">The Id of the Azure SQL Database.</param>
+        /// <param name="earliestRestoreDate">The recovery period start date
+        /// of the Azure SQL Database. This records the start date and time
+        /// when recovery is available for this Azure SQL Database (ISO8601
+        /// format).</param>
+        /// <param name="createMode">Specifies the type of database to create.
+        /// Possible values include: 'Copy', 'Default',
+        /// 'NonReadableSecondary', 'OnlineSecondary', 'PointInTimeRestore',
+        /// 'Recovery', 'Restore'</param>
+        /// <param name="sourceDatabaseId">Conditional.  Specifies the
+        /// resource Id of the source database.  If createMode is not set to
+        /// Default, then this value must be specified.</param>
+        /// <param name="edition">The edition of the Azure SQL Database.  The
+        /// DatabaseEditions enumeration contains all the valid editions.
+        /// Possible values include: 'Web', 'Business', 'Basic', 'Standard',
+        /// 'Premium', 'Free', 'Stretch', 'DataWarehouse'</param>
+        /// <param name="maxSizeBytes">The max size of the Azure SQL Database
+        /// expressed in bytes. Note: Only the following sizes are supported
+        /// (in addition to limitations being placed on each edition): { 100
+        /// MB | 500 MB |1 GB | 5 GB | 10 GB | 20 GB | 30 GB … 150 GB | 200
+        /// GB … 500 GB }</param>
```

Comment:
This list of max sizes should not be here. We should replace it with a suggestion to call the /capabilities API to get a list of max sizes per edition/slo/pool/dtu that are available.